api/src/main/resources/workbench-api.yaml

```diff
@@ -9795,11 +9799,18 @@ definitions:
       professionalUrl:
         type: string
         description: User's URL at primary place of work.
+      identityBypassTime:
```

Comment:
Is it a problem to send the reporting user more data than they are expecting? I know that we should coordinate with them when removing something.